conf/tasks/Makefile.gke

```diff
@@ -234,7 +234,7 @@ gke/destroy/bucket:
 ## Deploy helm tiller with service account
 gke/deploy/helm:
 	@echo "Deploying kubernetes resources..."
-	@kubectl create serviceaccount --namespace kube-system tiller 2> /dev/null || echo "Kubernetes already appears to be deployed."
```

Comment:
Ignoring the error output seems like a generally good thing. Should we even have the `|| echo` command? Maybe we should just fail here.

Come to think of it, could be nice to have a way to clean-up failures at this point.

Reply:
Yeah, I'm in favor of failing, too. I'll try removing the `||` and the `2>` and see if the output to the terminal is egregious.

Cleanup is a great idea and I've opened #274 to keep track of it.